### PR TITLE
Added affinity option to helm charts

### LINKED ISSUE #329: Missing affinity options for helm charts
I'm using datashim to connect to minio s3 bucket with access key and secret on AKS cluster with linux and windows nodes. due to missing affinity configuration on the Daemonsets for csi-nfs-chart & csi-s3-chart they attach to the windows node and fail to deploy.

is it possible for me to add PR to address this issue ?

### PR DESCRIPTION
**Pull request by muandane**
Added affinity options to charts for multi-cluster deployment cases (win + linux) 

fixes #329

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ global:
   arch: "amd64"
   type: "k8s"
   namespaceYaml: true
-
+  affinity: {}
```
Comment (srikumar003) [inline chart/values.yaml:7]: Please provide an example in a comment block on how to set the affinity values

**Review by srikumar003 — CHANGES_REQUESTED** [review]
Above line 7, please include a comment block with an example on how to set the affinity values

File: chart/values.yaml (hunk 1 above)
Comment (muandane) [reply chart/values.yaml:17]: is this acceptable as an example ?

**Comment by srikumar003** [comment]
approved. Thanks for your PR!

**Pull request merged**